geonode/geoserver/geofence.py

```diff
@@ -128,11 +229,11 @@ def __init__(self, baseurl: str, username: str, pw: str) -> None:
 
     def invalidate_cache(self):
         r = requests.put(
-            f'{self.baseurl}rest/geofence/ruleCache/invalidate',
+            f'{self.baseurl.rstrip("/")}/rest/geofence/ruleCache/invalidate',
```

Comment:
this change (and next similar ones) was better implemented in the next geofence PR.
It will be replaced